bytecode-compatibility-transformer/src/test/java/org/jenkinsci/bytecode/CompatibilityTest.java

```diff
@@ -84,8 +84,10 @@ private AntClassLoader createClassLoader(String v) throws IOException {
             }
         };
 
-        cl.addPathComponent(new File("target/test-classes/"+ v));
-        cl.addPathComponent(new File("target/test-classes/client"));
+        File p = new File("../test-client/target");
+        System.out.println("looking " + p.getAbsolutePath() + "; exists: " + p.exists());
+        cl.addPathComponent(new File("../test-" + v + "/target"));
```

Comment:
The v1 tests pass if you add the jars explicitly instead of just the folders, but the v2 tests fail, probably because `test-v2/target` does not contain a jar file, which happens because test-v2 comes after bytecode-compatibility-transformer module in the reactor order because it depends on that module.

Reply:
yes, Likely I will need to move tests to the ~~third~~ fifth module... hmm.. ok

Reply:
@oleg-nenashev I am already on that, should have a PR for your PR shortly.

Reply:
PR is up: oleg-nenashev/bytecode-compatibility-transformer#1